content/posts/messaging-infrastructure.md

```diff
@@ -0,0 +1,80 @@
+# Messaging infrastructure
+
+So far we've seen message de-duplication is not a simple topic and that *idempotency* is not a useful term then talking about distributed systems in which messages can get duplicated and reordered. Since we are sending our readers down the path of complexity we should at least discuss briefly what are the alternatives.
```

Comment:
We removed reference to idempotency in previous posts so it might not be clear here. I suggest to remove it.

Reply:
The section is called `Messaging infrastructue' - should we define it here? It should be clear that we mean the endpoint-side infrastructure more than the message broker.